apps/video/js/view.js

```diff
@@ -565,4 +591,4 @@ navigator.mozSetMessageHandler('activity', function viewVideo(activity) {
     dom.spinnerOverlay.classList.add('hidden');
     setDisabled(dom.playerView, false);
   }
-});
+})();
```

Comment:
Its a total nit, but the convention I follow is to use `())` instead of `)()`
Also note that it would be fine to not use the init() function at all and just have all of this in the global scope. I assume that it is probably simpler to do it this way, though.